### PR TITLE
Fix/dashboard circles

### PR DESCRIPTION
**Pull request by NanaKay007**
@phirework  This is a fix for the dashboard circles bug. Did exactly as you recommended :)

### CODE REVIEW

File: web/locales/en/messages.ftl [hunk 1]
```diff
@@ -731,6 +731,7 @@ validations = { $count ->
 ## Dashboard
 your-languages = Your Languages
 toward-next-goal = Toward next goal
+goal-reached = goal reached
```
Comment (phirework) [inline web/locales/en/messages.ftl:734]: Can you capitalize "goal" here?

**Comment by phirework** [comment]
Thanks for making those changes! Other than capitalizing "goal reached" this is good to go.

**Comment by NanaKay007** [comment]
grrrr... sorry. All fixed now

**Comment by phirework** [comment]
Thank you!

**Pull request merged**